inference_schema/parameter_types/standard_py_parameter_type.py

```diff
@@ -40,6 +40,21 @@ def __init__(self, sample_input):
                 if issubclass(type(data), AbstractParameterType):
                     self.sample_data_type_list.append(data)
 
+    def _get_supported_versions(self):
+        supported_list = ['3.0', '3.1']
```

Comment:
Is there a reason you chose strings over floats here? not sure it matters much either way, just curious

Reply:
Because "3.0" isn't as semantically meaningful as a float. If we need to do math with them, then we can change it